lib/shopify_api/resources/graphql.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'graphql/client'
+require 'graphql/client/http'
+
+module ShopifyAPI
+  # GraphQL API.
+  class GraphQL
+    def initialize
+      uri = Base.site.dup
+      uri.path = '/admin/api/graphql.json'
+      @client = ::GraphQL::Client::HTTP.new(uri.to_s) do
```

Comment:
Wouldn't this code also need to pass in the `X-Shopify-Storefront-Access-Token` header? I don't see where that is configured here at all.

Reply:
The storefront-access-token header is only necessary for the storefront API (https://help.shopify.com/api/custom-storefronts/storefront-api). This PR is for the admin API.

Reply:
Should @client here be @http?

Reply:
I think so. It doesn't affect the correctness of the code but it would be less confusing.